packages/core/src/awsService/appBuilder/serverlessLand/wizard.ts

```diff
@@ -22,160 +24,136 @@ export interface CreateServerlessLandWizardForm {
     iac: string
 }
 
-/**
- * Wizard for creating Serverless Land applications
- * Guides users through the project creation process
- */
-export class CreateServerlessLandWizard extends Wizard<CreateServerlessLandWizardForm> {
-    private metadataManager: MetadataManager
+async function loadMetadata(): Promise<MetadataManager> {
+    const metadataManager = MetadataManager.getInstance()
```

Comment:
does it make sense for this to live inside of the metadataManager? Or do you envision the metadata manager loading multiple different types of metadata

Reply:
The project will utilize a single metadata file. This file is included to provide the necessary path for accessing the metadata within the `wizard.ts` file. This serves as a call and initialization for the functionality. The metadata is then loaded through a separate `loadMetadata()` function located in the `metadataManager.ts` file.